plugins/obs-ffmpeg/ffmpeg-mux/ffmpeg-mux.c

```diff
@@ -200,6 +210,48 @@ static bool get_audio_params(struct audio_params *audio, int *argc,
 	return true;
 }
 
+static void ffmpeg_log_callback(void *param, int level, const char *format,
+				va_list args)
+{
+	const char *stream_key;
+	char out[4096];
+
+	struct dstr log_message = {0};
+	stream_key = global_stream_key;
+
+	vsnprintf(out, sizeof(out), format, args);
+	dstr_copy(&log_message, out);
+
+	if ((strcmp(stream_key, "") != 0) &&
+	    (dstr_find(&log_message, stream_key)))
+		dstr_replace(&log_message, stream_key, "{stream_key}");
```

Comment:
Do we need dstr_find? If stream_key is not present, does dstr_replace return the original string?

Reply:
Reread dstr_find and I think we don't need it. I was trying to be overly cautious.